be/src/storage/lake/rowset.cpp

```diff
@@ -38,9 +39,12 @@ Rowset::Rowset(TabletManager* tablet_mgr, int64_t tablet_id, const RowsetMetadat
           _tablet_id(tablet_id),
           _metadata(metadata),
           _index(index),
-          _tablet_schema(std::move(tablet_schema)) {}
+          _tablet_schema(std::move(tablet_schema)),
+          _parallel_load(false),
+          _compaction_segment_limit(0) {}
 
-Rowset::Rowset(TabletManager* tablet_mgr, TabletMetadataPtr tablet_metadata, int rowset_index)
+Rowset::Rowset(TabletManager* tablet_mgr, TabletMetadataPtr tablet_metadata, int rowset_index,
+               size_t compaction_segment_limit)
```

Comment:
init the _compaction_segment_limit in initialization list. so it is ensured to be inited to 0 for any case. And in the function body adjust it as needed or just leave as it.

Reply:
done